xclim/indicators/atmos/_temperature.py

```diff
@@ -781,11 +782,36 @@ def cfcheck(tas, snd=None):
     description="Temperature-based index used to estimate the development of corn crops. "
     "Corn growth occurs when the minimum and maximum daily temperature both exceeds "
     "specific thresholds : Tmin > {thresh_tasmin} and Tmax > {thresh_tasmax}.",
+    var_name="chu",
     cell_methods="",
     missing="skip",
     compute=indices.corn_heat_units,
 )
 
+huglin_index = Temp(
+    identifier="huglin_index",
+    units="",
+    long_name="Huglin heliothermal index (Summation of ((Tmin + Tmax)/2 - {thresh_tasmin}) * Latitude-based day-length"
+    "coefficient (`k`), for days between {start_date} and {end_date}).",
+    description="Heat-summation index for agroclimatic suitability estimation, developed specifically for viticulture. "
+    "Considers daily Tmin and Tmax with a base of {thresh_tasmin}, typically between 1 April and 30 September. "
+    "Integrates a day-length coefficient calculation for higher latitudes.",
+    cell_methods="",
+    comment="Metric originally published in Huglin (1978). Day-length coefficient based on Hall & Jones (2010)",
+    var_name="hi",
+    compute=wrapped_partial(
+        indices.huglin_index,
+        method="jones",
+        suggested=dict(
+            thresh_tasmin="10 degC",
+            start_date="04-01",
+            end_date="10-01",
+            lat=_empty,
+        ),
```

Comment:
```suggestion
```

This is unnecessary, as those are _already_ the default of the indice!

Reply:
Good to know!